site/docs/guidelines/vertical-rhythm-and-baseline-grid.mdx

```diff
@@ -22,29 +23,13 @@ In order for visual rhythm to work, not only do elements have to take up the sam
-* When adding text to your component or interface, we have SASS mixins that set the correct font family, font size, and position so that the text will sit on the baseline grid. See [typography](/guidelines/typography/) for details on these mixins.
-* If you need to use these mixins, but do not want the baseline grid adjustment, you can also use the `ca-inherit-baseline` mixin.
-* Because our mixins shift the element down on the page, we need to create space in the layout for the element to shift into. For this we have a `ca-type-block` mixin that creates the whitespace necessary for the contained text to shift “into”. That is, it creates the 1/2-grid `padding-bottom` on a container for the text to have room to be shifted “into”.
-    * Watch [“Teaching CSS to talk like a designer”](https://www.youtube.com/watch?v=TGHbkTGVqoU)
-    * Watch from 50:24 in the video recording of this meeting: [internal link](https://cultureamp.slack.com/files/U04S9RRAC/F8XPHLQ13/Front_End_Practice_Meeting_____2018-01-24)
-    * Watch video recording of this Brown Bag (which covered scrollbar issues and the work that we were doing on our baseline grid and vertical rhythm): [internal video link](https://cultureamp.zoom.us/recording/share/bqOJujzbx7saKe2366y4XBYVKUXSiDMJ1bgLchDFH0awIumekTziMw)
-    * We use `position: relative` and `top`. It doesn’t create a new _stacking_ context, but does create a new _positioning_ context.
-    * Previously, `translateY` caused issues with stacking context, and breaking `z-index` rules, so dropdown menus and others started becoming unruly to manage.
-    * With the normal baseline that comes from the `ca-type-inter-x` styles, padding on a box surrounding some text results in, visually, the box having reduced padding on the bottom even though the `padding-top` and `padding-bottom` values are exactly the same
-
-<iframe style="border: none;" width="744" height="450" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2Fe3TyAOZKfLg9cA01Iy1Mgm%2FKaizen-Site-embed%3Fnode-id%3D1273%253A23079" allowfullscreen></iframe>
-
-* That padding is there because the button and its label is shifted down onto the baseline grid, so it needs guaranteed whitespace to shift into.
-* All buttons are shifted, so that if you have icon buttons and text buttons (and even plain old text) they are well aligned
+* We previously used the [Basekick/baseline shift approach](https://github.com/michaeltaranto/basekick) but abandoned it because of the complexities it caused in implementation. Consumers often needed to be aware of what Basekick was and how it worked if they needed to make seemingly minute alterations. Now that Figma accounts for line height in a similar way to what the web does we've deprecated baseline shift. If you'd like to read more about what baseline shift was trying to solve, see [this Figma article](https://www.figma.com/blog/line-height-changes/).
```

Comment:
Maybe we could add some information around spacing components with `Box` and the spacing tokens instead?

Reply:
Good shout. Will add that in!

Reply:
How's this as another point below @ActuallyACat ?
>* Kaizen now uses the `<Box>` component to handle spacing, which is flexible enough to handle almost all your spacing needs without you having to reach for CSS.

Reply:
That looks good to me!

Reply:
@ActuallyACat would love a tick on this when you have a chance :)